playbooks/roles/aws/tasks/main.yml

```
@@ -124,5 +124,5 @@
     mode: "0644"
   with_items: "{{ motd_files_exist.results }}"
   when:
-    - vagrant_home_dir.stat.exists == False and ansible_distribution in common_debian_variants and item.stat.exists
     - cloud_provider == "aws"
+    - vagrant_home_dir.stat.exists == False and ansible_distribution in common_debian_variants and item.stat.exists
```

Maybe this

```suggestion
    - ansible_distribution in common_debian_variants and item.stat.exists
```

Vagrant is no more as of Hawthorn.